video/out/mac/title_bar.swift

```diff
@@ -91,34 +91,54 @@ class TitleBar: NSVisualEffectView {
         common.window?.isMoving = false
     }
 
-    func set(appearance: Any) {
-        if appearance is Int {
-            window?.appearance = appearanceFrom(string: String(appearance as? Int ?? 0))
-        } else {
-            window?.appearance = appearanceFrom(string: appearance as? String ?? "auto")
-        }
+    func set(appearance: Int32) {
+        window?.appearance = { switch appearance {
+            case 1: return NSAppearance(named: .aqua)
```

Comment:
Do we have to have this hardcoded mapping, isn't there better way? What does the int represent in fact?

Reply:
Yes, that's what I meant to have it defined once in one place.

Reply:
btw i didn't include this change here, because i wanted to try something else because i didn't like this mapping that much either.